services/prometheus/prometheus.go

```diff
@@ -129,7 +129,7 @@ func (svc *Service) reload() error {
 	}
 	defer resp.Body.Close() //nolint:errcheck
 
-	if resp.StatusCode == 200 {
+	if resp.StatusCode == 200 { //nolint:gomnd
```

Comment:
Instead of this we could use http.StatusOK

Reply:
I had a discussion about that with AlekSi a few times. His point that `200` easier and more native (or something like that) and `http.StatusOK` is redundant. I do not fully agree, but follow common approaches in our codebase.